src/components/JobDetailPage.tsx

```diff
@@ -83,7 +117,46 @@ const StyledTypographySQL = withStyles({
   }
 })(Typography)
 
+const StyledExpandButton = withStyles({
+  root: {
+    transform: 'rotate(45deg)',
+    position: 'absolute',
+    right: 0,
+    bottom: 0,
+    backgroundColor: 'white',
+    cursor: 'pointer'
+  }
+})(OpenWithSharpIcon)
+
+const StyledCloseIcon = withStyles({
+  root: {
+    position: 'absolute',
+    right: '2rem',
+    top: '1rem',
```

Comment:
A minor thing, but the icon to close the modal is sitting just above the frame of the modal
<img width="140" alt="Screen Shot 2019-12-04 at 1 07 51 PM" src="https://user-images.githubusercontent.com/1332789/70168582-2aabdb00-1697-11ea-98a1-6e220a093830.png">
Actually it looks like there is a black border at the top of the modal.
<img width="1687" alt="Screen Shot 2019-12-04 at 1 03 23 PM" src="https://user-images.githubusercontent.com/1332789/70168730-6777d200-1697-11ea-9cea-993339268fe1.png">
Change line 135 to `    top: '2rem',` to get rid of the overlap?